src/types/index.d.ts

```diff
@@ -180,3 +180,28 @@ export type FilterExpressionRange = {
 };
 
 export type FilterExpressionRangeValue = ['-inf' | number, 'inf' | number];
+
+export interface Item extends Record<string, any> {
+  value: string;
+  is_slotted: boolean;
+  labels: Record<string, unknown>;
+  matched_terms: string[];
+  data?: ItemData;
+}
+
+export interface ItemData extends Record<string, any> {
+  url?: string;
+  id?: string;
+  image_url?: string;
+  group_ids?: string[];
```

Comment:
Can we add `groups` here as well?

Reply:
Added groups as an optional parameter as discussed